### DISCUSSION TITLE
Dialog light dismiss doesn't raise OnDialogResult

### DISCUSSION THREAD

**Discussion by tlmii** [microsoft/fluentui-blazor#1597] [Q&A]
It feels odd to me that light dismiss (clicking outside the dialog to dismiss it) doesn't raise the OnDialogResult(Cancel) callback the way the dismiss (x) button does. Is there a technical/standards/compat reason for that I might not be thinking of?

**Comment by dvoituron** [comment]
That's a good question. I also think we could engage `OnDialogResult(Cancel)`, without even having a Breaking Change effect with the existing code.
